test/desdemona/query_test.clj

```diff
@@ -132,16 +166,26 @@
        [{:ip "10.0.0.2"    ;; ip clause succeeded
          :type "ingress"}]])))
 
-(deftest logic-query-tests
+(deftest run-logic-query-tests
   (are [query results] (= results (#'q/run-logic-query query events))
     'l/fail
     []
 
-    '(l/featurec x {:ip "10.0.0.1"})
+    (list clojure.core.logic/featurec
+          (#'q/free-sym 'x)
+          {:ip "10.0.0.1"})
+    [[{:ip "10.0.0.1"}]]
+
+    (list clojure.core.logic/featurec
+          (#'q/free-sym 'y)
+          {:ip "10.0.0.1"})
```

Comment:
I'm ok with this running core.logic queries directly instead of requiring dsl->logic to create the query itself. It seems consistent with what `run-logic-query` expects to take as arguments.